### DISCUSSION TITLE
Demodulating and decoding Signals generated by the modules on SDR based application

### DISCUSSION THREAD

**Discussion by rnbokade** [jgromes/RadioLib#220] [Ideas]
For example, AX.25 on AFSK in some of the modules like the SX127x series can be demodulated as FM, and audio generated can be sent to sound modem or direwolf to get the packet data out. While in the case of the SX126x series where there is a lack of direct modulation, the signal needs to be demodulated as LSB and then audio is to be passed to soundmodem. There is a lot of confusion like that about the signals. Can you specify that in SDR_demode.md or something

**Comment by jgromes** [comment]
There's a wiki page for it: https://github.com/jgromes/RadioLib/wiki/SDR-Digital-Modes-Guide
A bit bare at the moment, but I will add a couple pictures in the next few days.
